tests/testsuite/build_script.rs

```diff
@@ -3707,7 +3707,7 @@ fn panic_abort_with_build_scripts() {
     p.root().join("target").rm_rf();
 
     p.cargo("test --release -v")
-        .with_stderr_does_not_contain("[..]panic[..]")
+        .with_stderr_does_not_contain("[..]panic=abort[..]")
```

Comment:
Are the test changes ones that would make sense outside of cargo-test-macro changes?  If so, let's do them in a separate PR or at least a separate commit.  That way, bisect, revert, etc work on the right level.

Reply:
I opened #11745 to move the test changes out of this PR